crates/l2/proposer/mod.rs

```diff
@@ -233,11 +241,27 @@ impl Proposer {
         keccak(block.encode_to_vec())
     }
 
-    pub async fn send_commitment(&self, commitment: H256) -> Result<H256, ProposerError> {
+    pub async fn send_commitment(
+        &self,
+        block_number: u64,
+        new_l2_state_root: H256,
+        withdrawal_logs_merkle_root: H256,
+    ) -> Result<H256, ProposerError> {
         info!("Sending commitment");
         let mut calldata = Vec::with_capacity(68);
         calldata.extend(COMMIT_FUNCTION_SELECTOR);
-        calldata.extend(commitment.0);
+        let mut block_number_bytes = [0_u8; 32];
+        U256::from(block_number).to_big_endian(&mut block_number_bytes);
+        calldata.extend(block_number_bytes);
+        calldata.extend(new_l2_state_root.0);
+        calldata.extend(withdrawal_logs_merkle_root.0);
+
+        println!("BLOCK NUMBER: {:?}", hex::encode(block_number_bytes));
+        println!("NEW L2 STATE ROOT: {:?}", hex::encode(new_l2_state_root));
+        println!(
+            "WITHDRAWAL LOGS MERKLE ROOT: {:?}",
+            hex::encode(withdrawal_logs_merkle_root)
+        );
```

Comment:
Is this intended to stay? In that case we should replace them with `tracing`s

Reply:
Done [5249393](https://github.com/lambdaclass/lambda_ethereum_rust/pull/950/commits/52493936e83ce8fd9125efb01db889d79a9fbd1e)